easybuild/easyconfigs/a/AlphaPulldown/AlphaPulldown-0.30.4_fix-deps.patch

```diff
@@ -0,0 +1,34 @@
+# making AlphaPulldown's dependencies less strict
+diff -ruN alphapulldown-0.30.4_orig/setup.cfg alphapulldown-0.30.4/setup.cfg
+--- alphapulldown-0.30.4_orig/setup.cfg	2023-04-05 13:59:25.000000000 +0100
++++ alphapulldown-0.30.4/setup.cfg	2023-04-06 14:10:45.424577468 +0100
+@@ -17,19 +17,19 @@
+ include_package_data = True
+ python_requires = ==3.8.*
+ install_requires = 
+-	absl-py >= 0.13.0
+-	dm-haiku==0.0.9
++	absl-py >= 0.10.0
++	dm-haiku==0.0.4
+ 	dm-tree==0.1.6
```

Comment:
This needs to be tweaked to `0.1.5` if we use a proper dependency for `dm-tree`

Reply:
resolved